airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -43,7 +43,7 @@ class SqoopHook(BaseHook, LoggingMixin):
     :param verbose: Set sqoop to verbose.
     :type verbose: bool
     :param num_mappers: Number of map tasks to import in parallel.
-    :type num_mappers: str
+    :type num_mappers: int
```

Comment:
should have users pass it in as an `int` type because it will be converted to str later on, [here](https://github.com/apache/incubator-airflow/pull/2703/files#diff-8e77f042c2e060bbfd60828431a91e9bL131). This data type makes most sense since it's the number of mappers. Logic for converting it should be abstract from us

Reply:
added test to make sure if `num_mappers` is `None` then that option doesn't get added to Sqoop command

Reply:
Great change.